rust/cloud-storage/rpc/axum_rpc/.helix/languages.toml

```diff
@@ -0,0 +1,5 @@
+[[language]]
```

Comment:
can you add this file to .gitignore.

this is in the same vein as committing a .idea folder